docs/admin/install/install.rst

```diff
@@ -191,55 +95,113 @@ With the key and configuration available in ``dom0``, you're ready to set up Sec
+      sdw-admin --configure
 
-.. note::
+  Follow the command prompts to complete submission key import. 
 
-   You can find the values for the **hidserv.*** fields in the ``/tmp/journalist.txt`` file that you created in ``dom0`` earlier.
-   The file will be formatted as follows:
+  .. note::
+    If there are multiple keys present on the device, ``sdw-admin --configure`` will print the fingerprints of those keys for you to select which to use as the submission private key. You can open ``<source interface address>.onion/metadata`` in Tor Browser on another network-connected computer to check the correct key fingerprint used by your SecureDrop instance.
 
-   .. code-block:: none
+- Once the submission key import is complete, in the ``vault`` file manager, right-click on the **TailsData** sidebar entry, then select **Unmount** and disconnect the SVS USB.
 
-     ONIONADDRESS:descriptor:x25519:AUTHTOKEN
+- If you were prompted for a passphrase during import, you will now need to remove the passphrase on ``sd-journalist.sec``. See :doc:`/admin/reference/removing_gpg_passphrase`.
 
-- Verify that the configuration is valid using the command below in the ``dom0`` terminal:
+.. _copy_journalist:
 
-  .. code-block:: sh
+Import *Journalist Interface* details
+-------------------------------------
+
+SecureDrop Workstation connects to your SecureDrop instance's API via the *Journalist Interface*. In order to do so, it will need the *Journalist Interface* address and authentication info. As the clipboard from another VM cannot be copied into ``dom0`` directly, follow these steps to copy the file into place:
+
+- Locate an *Admin Workstation* or *Journalist Workstation* USB drive. Both hold the address and authentication info for the *Journalist Interface*; if you also want to copy the journalist user's password database, use the *Journalist Workstation* USB drive.
 
-    sdw-admin --validate
+- Connect the USB drive to a USB port on the Qubes computer, then use the devices widget in the upper right panel to attach it to the ``vault`` VM. There will be 3 listings for the USB in the widget: one for the base USB, one for the Tails partition on the USB, labeled ``Tails``, and a 3rd unlabeled listing, for the persistent volume. Choose the third listing.
+
+- In the the ``vault`` file manager, select the persistent volume's listing in the lower left sidebar. It will be named ``N GB encrypted``, where N is the size of the persistent volume. Enter the persistent volume passphrase to unlock and mount it. When prompted, select the option to **Forget password immediately**.
 
-If you encounter a validation error due to a password-protected GPG key, see :doc:`/admin/reference/removing_gpg_passphrase`.
+- In the ``dom0`` terminal, proceed with the next import step of the ``sdw-admin`` command or re-run 
+
+  .. code-block:: sh 
+
+      sdw-admin --configure 
+
+  The command will print out the imported Journalist Interface details to confirm before proceeding.
```

Comment:
```suggestion
  The command will print out the imported *Journalist Interface* details to confirm before proceeding.
```

Glossary terms should be italicized - I think this one was just missed, all the others I've seen so far LGTM